test/test_models.py

```diff
@@ -40,22 +52,32 @@ def get_available_video_models():
 # they are not yet supported in JIT.
 script_test_models = [
     "deeplabv3_resnet101",
-    "mobilenet_v2",
-    "resnext50_32x4d",
     "fcn_resnet101",
-    "googlenet",
-    "densenet121",
-    "resnet18",
-    "alexnet",
-    "shufflenet_v2_x1_0",
-    "squeezenet1_0",
-    "vgg11",
-    "inception_v3",
     'r3d_18',
 ]
 
 
 class ModelTester(TestCase):
+
+    # create random tensor with given shape using synced RNG state
+    # caching because these tests take pretty long already (instantiating models and all)
+    TEST_INPUTS = {}
+
+    def _get_test_input(self, shape=STANDARD_INPUT_SHAPE):
```

Comment:
Did you test that this had any noticeable perf improvement?

Reply:
The speed-up is not huge - about a second on a test suite that takes about 200s. It's also not a huge memory hit, since currently only two modestly-sized input tensors are being created. If it becomes an issue, I'm happy to make needed changes.